store/storetest/team_store.go

```diff
@@ -1346,9 +1377,9 @@ func testGetTeamsByScheme(t *testing.T, ss store.Store) {
 		Type:        model.TEAM_OPEN,
 	}
 
-	_ = (<-ss.Team().Save(t1)).Data.(*model.Team)
-	_ = (<-ss.Team().Save(t2)).Data.(*model.Team)
-	_ = (<-ss.Team().Save(t3)).Data.(*model.Team)
+	ss.Team().Save(t1)
```

Comment:
Would you please check the returned errors here?

```suggestion
	err := ss.Team().Save(t1)
	require.Nil(t, err)
```

Reply:
Gently reminder about this comment :win